web/controllers/courseController/index.ts

```diff
@@ -2334,34 +2334,32 @@ async function groupMail(student: Student, courseId: number, subcourseId: number
 }
 
 /**
- * @api {POST} /course/:id/meeting/join JoinCourseMeeting
+ * @api {GET} /course/:id/subcourse/:subid/meeting GetCourseMeeting
  * @apiVersion 1.1.0
  * @apiDescription
- * Joins the BBB-Meeting for a given course
+ * Get the BBB-Meeting for a given subcourse
  *
- * This endpoint allows joining the BBB-Meeting of a course.
- * If the user is the instructor of the course the Meeting gets created with this call.
- * The other participants can only join after the instructor created the meeting with this endpoint
+ * This endpoint provides the BBB-Meeting of a subcourse.
  *
- * @apiName JoinCourseMeeting
+ * @apiParam (URL Parameter) {int} id ID of the main course
+ * @apiParam (URL Parameter) {int} subid ID of the subcourse
+ *
+ * @apiName GetCourseMeeting
  * @apiGroup Courses
  *
  * @apiUse Authentication
- * @apiUse Course
- *
- * @apiParam (JSON Body) {int} subcourseId ID of the subcourse of the course that should be joined
  *
  * @apiExample {curl} Curl
- * curl -k -i -X POST -H "Token: <AUTHTOKEN>" https://api.corona-school.de/api/course/<ID>/meeting/join
+ * curl -k -i -X GET -H "Token: <AUTHTOKEN>" https://api.corona-school.de/api/course/<ID>/subcourse/<ID>/meeting
  *
  * @apiUse StatusOk
  * @apiUse StatusBadRequest
```

Comment:
I think what the API returns here is again missing. I think that was accidentally removed https://github.com/corona-school/backend/pull/125/commits/01abe7dc56d0d5d6b08785ed7018d5c33234bb8c#diff-41dab649f245dd7fb8b0ea1406ee7122L361-L366